controllers/staff.js

```diff
@@ -81,6 +82,26 @@ FROM (Orders natural join OrderRestaurantPair join RidToName using (riderId) joi
 WHERE rid = $1
 ORDER BY orderTime DESC, deliveredTime DESC;`});
 
+const psGetRestaurantPromos = new PS({ name: 'get-promos', text: `SELECT pid, points, startDate, endDate, percentOff, minSpending, monthsWithNoOrders from Promotions natural join RestaurantPromos WHERE rid = $1 ORDER BY pid;` });
+
+const psInsertPromo = new PS({ name: 'get-promo-by-id', text:
+`
+INSERT into Promotions (startDate, endDate, points, percentOff, minSpending, monthsWithNoOrders) values
+($1, $2, $3, $4, $5, $6) RETURNING pid;
+`});
+
+const psInsertRestaurantPromo = new PS({ name: 'insert-rest-promo', text: `INSERT into RestaurantPromos (rid, pid) values ($1, $2);` });
+
+const psEditRestaurantPromos = new PS({ name: 'edit-promo', text:
+`
+UPDATE Promotions
+SET
+startDate = $2, endDate = $3, points = $4, percentOff = $5, minSpending = $6, monthsWithNoOrders  = $7
+WHERE pid = $1;
+`});
+
+const psDeleteRestaurantPromos = new PS({ name: 'delete-promo', text: 'DELETE FROM Promotions WHERE pid = $1;' });
```

Comment:
This deletes from promotions, allowing restaurant to delete global promotion

Reply:
fixed